server/game/game.js

```diff
@@ -235,17 +249,26 @@ Game.prototype.spyNextMove = function(player, hint, hintNum) {
     throw new Error('Invalid number of hints');
 
   this.gameState.hint = hint;
-  this.gameState.guessNum = hintNum + 1;
+  this.gameState.guessNum = hintNum + this.gameState.guessNum;
```

Comment:
When I was testing this, I had to run a parseInt on the hintNum variable (with some conditionals) since it was treating this variable as a string otherwise.

Reply:
Resolved this issue via adding a condition to check hintNum. Throw an error if it is not a number type value.